internal/testrunner/runners/system/servicedeployer/kubernetes.go

```diff
@@ -132,15 +132,17 @@ func (ksd KubernetesServiceDeployer) installCustomDefinitions() error {
 
 var _ ServiceDeployer = new(KubernetesServiceDeployer)
 
-func findKubernetesDefinitions(definitionsDir string) ([]string, error) {
-	files, err := filepath.Glob(filepath.Join(definitionsDir, "*.yaml"))
-	if err != nil {
-		return nil, errors.Wrapf(err, "can't read definitions directory (path: %s)", definitionsDir)
+func findKubernetesDefinitions(definitionsDir string) (string, error) {
```

Comment:
what about `checkKubernetesDefinition(definitionsDir string) (error)` since it just validates that directory either contains an .empty file or a kustomization.yaml.